code-studio/src/js/clientState.js

```diff
@@ -251,9 +376,9 @@ function recordVisualElementSeen(visualElementType, visualElementId) {
  * @param visualElementId
  */
 function hasSeenVisualElement(visualElementType, visualElementId) {
-  var elementSeenJson = sessionStorage.getItem(visualElementType) || '{}';
+  var elementSeenJson = getItem(visualElementType) || {};
   try {
-    var elementSeen = JSON.parse(elementSeenJson);
+    var elementSeen = elementSeenJson;
     return elementSeen[visualElementId] === true;
```

Comment:
Nit: I don't think this can throw anymore since we're no longer parsing JSON?  I think this can just be:
```javascript
function hasSeenVisualElement(visualElementType, visualElementId) {
  var elementSeen = getItem(visualElementType) || {};
  return elementSeen[visualElementId] === true;
}
```